deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: oadp-operator
           # Replace this with the built image name
-          image: "REPLACE_IMAGE"
+          image: "quay.io/spampatt/oadp-operator"
```

Comment:
We can leave this for now. I will work to get us autobuilds in quay tied to github and we can use that image here.

Reply:
Done